barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_group.test.cpp

```diff
@@ -1683,36 +1683,39 @@ TYPED_TEST(CycleGroupTest, TestMul)
             typename Group::Fr native_scalar = Group::Fr::random_element(&engine);
             auto expected_result = element * native_scalar;
 
-            // 1: add entry where point, scalar are witnesses
+            // 1: perform mul where point, scalar are witnesses
             point = (cycle_group_ct::from_witness(&builder, element));
             scalar = (cycle_group_ct::cycle_scalar::from_witness(&builder, native_scalar));
             point.set_origin_tag(submitted_value_origin_tag);
             scalar.set_origin_tag(challenge_origin_tag);
             result = point * scalar;
```

Comment:
This test was intending to perform a mul with different types of inputs but for all but the first case it wasnt actually performing the mul. Fixing this is what accounts for the gate diff below.